grpc.go

```diff
@@ -1465,6 +1590,14 @@ func (a *agentGRPC) OnlineCPUMem(ctx context.Context, req *pb.OnlineCPUMemReques
 	return emptyResp, a.onlineCPUMem(req)
 }
 
+func (a *agentGRPC) OfflineCPUMem(ctx context.Context, req *pb.OfflineCPUMemRequest) (*gpb.Empty, error) {
```

Comment:
Please could you add a test to `grpc_test.go` similar to `TestOnlineCPUMem()`.

Reply:
ok